src/VisualStudio/CSharp/Impl/Options/IntelliSenseOptionPageControl.xaml.cs

```diff
@@ -19,15 +18,15 @@ public IntelliSenseOptionPageControl(OptionStore optionStore) : base(optionStore
         {
             InitializeComponent();
 
-            BindToOption(Automatically_complete_statement_on_semicolon, CompleteStatementOptionsStorage.AutomaticallyCompleteStatementOnSemicolon);
+            BindToOption(Show_completion_list_after_a_character_is_typed, CompletionOptionsStorage.TriggerOnTypingLetters, LanguageNames.CSharp);
```

Comment:
I reorder the sequence here to match the order in xaml.
![image](https://github.com/dotnet/roslyn/assets/24360909/8517371b-fc16-4ffc-9ba1-a604582946f2)